source/Assets/Editor/DidomiPackageExporter.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+public class DidomiPackageExporter
+{
+    
+    /// <summary>
+    /// Export the unitypackage files for the plugin release
+    /// </summary>
+    [MenuItem("Didomi/Export Packages")]
+    static void ExportPackages()
+    {
+        ExportStandardPackage();
+        ExportNoDllPackage();
+    }
+
+    /// <summary>
+    /// Export the Didomi.unitypackage file for the plugin release
+    /// </summary>
+    static void ExportStandardPackage()
+    {
+        string[] exported = GetPathsToExport(noDll: false).ToArray();
+
+        try
+        {
+            AssetDatabase.ExportPackage(exported, "Didomi.unitypackage", ExportPackageOptions.Recurse);
+            Debug.Log("Standard package exported successfully");
+        }
+        catch (Exception ex)
+        {
+            Debug.LogErrorFormat("Error while exporting standard package: {0}", ex.Message);
+            throw ex;
+        }
+    }
+
+    /// <summary>
+    /// Export the Didomi-noDll.unitypackage file for the plugin release
+    /// </summary>
+    static void ExportNoDllPackage()
+    {
+        string[] exported = GetPathsToExport(noDll: true).ToArray();
+
+        try
+        {
+            AssetDatabase.ExportPackage(exported, "Didomi-noDll.unitypackage", ExportPackageOptions.Recurse);
+            Debug.Log("noDll package exported successfully");
+        }
+        catch (Exception ex)
+        {
+            Debug.LogErrorFormat("Error while exporting noDll package: {0}", ex.Message);
+            throw ex;
+        }
+    }
+
+    private static List<string> GetPathsToExport(bool noDll)
+    {
+        string path = Path.Combine(Application.dataPath, "Plugins" + Path.DirectorySeparatorChar + "Didomi");
+        Debug.LogFormat("Exporting package from {0}", path);
+
+        List<string> included = new List<string>();
```

Comment:
not sure if this version of C# for Unity supports it (I think it does b/c this should be at least 4.x), but you could declare this as `var included = new List<string>();` (see [Implicitly Type Variables](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/statements/declarations#implicitly-typed-local-variables)).